Src/ILGPU/Backends/PointerViews/LowerPointerViews.cs

```diff
@@ -107,10 +121,17 @@ public PointerViewLowering(Method.Builder builder)
                 pointer,
                 value.Offset);
 
+            var length = value.Length;
+            if (length.BasicValueType < BasicValueType.Int64)
```

Comment:
QUESTION: What is the purpose of comparing `length.BasicValueType < BasicValueType.Int64`?

Is this really just `length.BasicValueType != BasicValueType.Int64`? Or are `BasicValueType.FloatXX` allowed?

Reply:
Floats are not allowed; the idea was to cover all cases where `length` is not a 64-bit value. However, I also prefer `!=` in this context.